### PR TITLE
[UXE-5100] feat: add breakdown dashboard in bot manager advanced page

### PR DESCRIPTION
**Pull request by william-tome**
### What is the new behavior introduced by this PR?
- Add new dashboard (breakdown) to bot manager advanced
- Adjust file references
- Update tests
- Remove duplicate field in queries
- Update colors in map
- Refactor legend component in map chart
- Display new filters available in securityMetrics dataset

### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.
Screenshot 2024-09-24 at 16 42 32

### Does it have a link on Figma?
No
<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [x] Application responsiveness was tested to different screen sizes
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: src/modules/real-time-metrics/constants/help-center-urls.js [hunk 1]
```diff
@@ -78,19 +78,26 @@ const HELP_CENTER_URLS = {
...
+      topBotCaptchaPie:
+        '/real-time-metrics/bot-manager-advanced/overview/top-bot-captcha-pie-graph',
+      topBotClassification:
+        '/real-time-metrics/bot-manager-advanced/overview/top-bot-classifications',
       botActivityMap: '/real-time-metrics/bot-manager-advanced/overview/bot-activity-map'
+    },
+    botManagerBreakdown: {
+      impactedUrls: '/real-time-metrics/bot-manager-advanced/breakdown',
+      topIps: '/real-time-metrics/bot-manager-advanced/breakdown',
+      topImpactedUrls: '/real-time-metrics/bot-manager-advanced/breakdown'
```
Comment (MarianaAguilera) [inline src/modules/real-time-metrics/constants/help-center-urls.js:98]:
```suggestion
      impactedUrls: '/real-time-metrics/bot-manager-advanced/overview/impacted-urls',
      topIps: '/real-time-metrics/bot-manager-advanced/overview/top-ips',
      topImpactedUrls: '/real-time-metrics/bot-manager-advanced/overview/top-impacted-urls''
```

File: src/tests/modules/real-time-metrics/constants/reports.test.js [hunk 2]
```diff
@@ -1110,6 +1111,99 @@ describe('RealTimeMetricsModule', () => {
...
+          filters: {
+            metricEq: 'uniq_request_path',
+            datasetEq: 'bot_manager'
+          },
+          groupBy: [],
+          fields: [],
+          orderDirection: 'DESC',
+          dashboardId: '659868342290523846',
+          variationType: 'inverse',
+          helpCenterPath: '/real-time-metrics/bot-manager-advanced/breakdown'
```
Comment (MarianaAguilera) [inline src/tests/modules/real-time-metrics/constants/reports.test.js:1143]:
```suggestion
          helpCenterPath: '/real-time-metrics/bot-manager-advanced/overview/impacted-urls'
```

File: src/tests/modules/real-time-metrics/constants/reports.test.js [hunk 3]
```diff
@@ -1110,6 +1111,99 @@ describe('RealTimeMetricsModule', () => {
...
+            datasetEq: 'bot_manager',
+            metricEq: 'top_request_path'
+          },
+          limit: 10,
+          groupBy: ['dimension1'],
+          fields: ['dimension1'],
+          orderDirection: 'DESC',
+          dashboardId: '659868342290523846',
+          variationType: 'regular',
+          helpCenterPath: '/real-time-metrics/bot-manager-advanced/breakdown'
```
Comment (MarianaAguilera) [inline src/tests/modules/real-time-metrics/constants/reports.test.js:1205]:
```suggestion
          helpCenterPath: '/real-time-metrics/bot-manager-advanced/overview/top-impacted-urls'
```

File: src/modules/real-time-metrics/constants/reports-texts.js [hunk 4]
```diff
@@ -206,6 +206,17 @@ const REPORTS_TEXTS = {
       botActivityMap: {
         description: 'Sum of requests identified as bots, presented by the country of origin.'
       }
+    },
+    botManagerBreakdown: {
+      impactedUrls: {
+        description: 'TBD'
```
Comment (MarianaAguilera) [inline src/modules/real-time-metrics/constants/reports-texts.js:212]:
```suggestion
        description: 'Sum of the detected bot actions, broken down by the affected URLs.'
```

File: src/modules/real-time-metrics/constants/reports-texts.js [hunk 5]
```diff
@@ -206,6 +206,17 @@ const REPORTS_TEXTS = {
...
+      },
+      topIps: {
+        description: 'TBD'
```
Comment (MarianaAguilera) [inline src/modules/real-time-metrics/constants/reports-texts.js:215]:
```suggestion
        description: 'Sum of the detected bot actions, broken down by the highest-ranking IP addresses.'
```

File: src/modules/real-time-metrics/constants/reports-texts.js [hunk 6]
```diff
@@ -206,6 +206,17 @@ const REPORTS_TEXTS = {
...
+      },
+      topImpactedUrls: {
+        description: 'TBD'
```
Comment (MarianaAguilera) [inline src/modules/real-time-metrics/constants/reports-texts.js:218]:
```suggestion
        description: 'Sum of the detected bot actions, broken down by the most affected URLs.'
```

File: src/tests/modules/real-time-metrics/constants/reports.test.js [hunk 7]
```diff
@@ -1110,6 +1111,99 @@ describe('RealTimeMetricsModule', () => {
...
+        },
+        {
+          id: '1030427483148242',
+          chartOwner: 'azion',
+          label: 'Top Impacted URLs',
+          description: 'TBD',
```
Comment (MarianaAguilera) [inline src/tests/modules/real-time-metrics/constants/reports.test.js:1180]:
```suggestion
          description: 'Sum of the detected bot actions, broken down by the most affected URLs.',
```

**Review by pauloSF0 — APPROVED** [review]

File: src/modules/real-time-metrics/reports/convert-beholder-to-chart.js [hunk 8]
```diff
@@ -429,7 +441,11 @@ const formatBigNumbers = ({ report, data }) => {
   const aggregation = report.aggregationType
 
   const total = data[dataset].reduce((acc, current) => acc + current[aggregation || fieldName], 0)
-  const { unit, value } = formatDataUnit(total, report)
+  let { unit, value } = formatDataUnit(total, report)
+
+  if (report.id === '847143804009563421') {
+    unit = 'URLs'
+  }
```
Comment (HerbertJulio) [inline src/modules/real-time-metrics/reports/convert-beholder-to-chart.js:448]: Sugiro crair uma const para esse valor
```suggestion
const TARGET_REPORT_ID_URLS = '847143804009563421';
  if (report.id === TARGET_REPORT_ID_URLS) {
    unit = 'URLs'
  }
```

File: src/modules/real-time-metrics/reports/convert-beholder-to-chart.js [hunk 9]
```diff
@@ -330,6 +330,18 @@ const formatTsChartData = ({
...
+    seriesArray.forEach((serie) => {
+      // linter complains about ternary
+      if (serie[0] === 'false') {
+        serie[0] = 'Not Solved'
+      }
+      if (serie[0] === 'true') {
+        serie[0] = 'Solved'
+      }
+    })
+  }
```
[the remark this answers is not in the document]
Comment (william-tome) [reply src/modules/real-time-metrics/reports/convert-beholder-to-chart.js:344]: Deixei comentário. Linter rejeita essa abordagem

File: src/modules/real-time-metrics/reports/convert-beholder-to-chart.js (hunk 8 above)
Comment (william-tome) [reply src/modules/real-time-metrics/reports/convert-beholder-to-chart.js:448]: Só tem uma por enquanto. Caso haja mais, eu movo pra uma constante.

File: src/templates/graphs-card-block/components/map/utils/features-handler.js [hunk 10]
```diff
@@ -1,22 +1,24 @@
...
+      return `${rangeVariation}-limit4`
+    case value >= LIMITS.limit3:
+      return `${rangeVariation}-limit3`
+    case value >= LIMITS.limit2:
+      return `${rangeVariation}-limit2`
     default:
-      return `${rangeVariation}-low`
+      return `${rangeVariation}-limit1`
   }
 }
```
Comment (HerbertJulio) [inline src/templates/graphs-card-block/components/map/utils/features-handler.js:23]:
```suggestion
const rangeVariations = (value, rangeVariation) => {
  if (value > LIMITS.limit5) return `${rangeVariation}-limit5`;
  if (value >= LIMITS.limit4) return `${rangeVariation}-limit4`;
  if (value >= LIMITS.limit3) return `${rangeVariation}-limit3`;
  if (value >= LIMITS.limit2) return `${rangeVariation}-limit2`;
  return `${rangeVariation}-limit1`;
};
```

File: src/modules/real-time-metrics/reports/convert-beholder-to-chart.js (hunk 9 above)
[the remark this answers is not in the document]
Comment (william-tome) [reply src/modules/real-time-metrics/reports/convert-beholder-to-chart.js:344]: Vou desabilitar a regra aqui e corrigir no proximo PR.

File: src/templates/graphs-card-block/components/map/utils/features-handler.js (hunk 10 above)
Comment (william-tome) [reply src/templates/graphs-card-block/components/map/utils/features-handler.js:23]: Vou corrigi no proximo PR.

**Pull request merged**